### ISSUE TITLE
[BUG] Conflict between Next.js `lint` and VSCode errors

### ISSUE THREAD

**Issue by EliasVal** [francoismassart/eslint-plugin-tailwindcss#350]
**Describe the bug**
There's a conflict between VSCode/ESlint reporting the error in the editor (Editor reports as OK) and `next lint` (reports `tailwindcss/classnames-order`

**To Reproduce**
1. Configure `tailwindcss/classnames-order` to `error`
2. See what your editor reports for ESlint
3. Run `next lint` to check for errors

**Expected behavior**
Parity between Editor and `next lint`

**Screenshots**

**Environment (please complete the following information):**
 - OS: Win 11
 - Softwares + version used:
     -  VSCode v1.91.0
     -  Node.js v20.12.2
     -  Next.js v15.0.0-canary.53
     -  eslint-plugin-tailwindcss v3.17.4

**Additional context**

### THIS PROJECT IS A MONOREPO!!

Original Line (OK by VSCode):
```tsx
<ul
  className={`hover:bg-lightest-gray w-full rounded-full ${htmlDir === 'rtl' ? 'rounded-l-none' : 'rounded-r-none'} hover:bg-lightest-gray`}
>
```

Suggested fix (By `next lint`)
```tsx
<ul
  className={`w-full rounded-full hover:bg-lightest-gray ${htmlDir === 'rtl' ? 'rounded-l-none' : 'rounded-r-none'} hover:bg-lightest-gray`}
>
```

**eslint config file or live demo**
```ts
/* configs/eslint-config-custom/index.js */
/* (simplified) */
module.exports = {
  "settings": {
    "react": {
      "version": "detect"
    }
  },
  "plugins": [
    "react",
    "@typescript-eslint",
    "simple-import-sort",
    "jsx-a11y",
    "@next/eslint-plugin-next",
    "@stylistic",
    "unused-imports",
    "eslint-plugin-react-compiler"
  ],
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "ecmaVersion": "latest",
    "sourceType": "module",
    "project": "../../tsconfig.json"
  },
  "extends": [
    "plugin:react/recommended",
    "plugin:@typescript-eslint/recommended",
    "plugin:@next/next/recommended",
    "plugin:jsx-a11y/recommended",
    "plugin:tailwindcss/recommended",
    "plugin:deprecation/recommended"
  ],
  "rules": {
    "tailwindcss/no-custom-classname": "off",
    "tailwindcss/classnames-order": "error",
  }
}
```

```ts
/* apps/web/.eslintrc.json */
{
  "root": true,
  "extends": ["custom"],
  "settings": {
    "next": {
      "rootDir": "./"
    }
  },
  "parserOptions": {
    "project": true
  }
}
```

**Comment by gabrielzevedo** [comment]
[This suggestion](https://github.com/francoismassart/eslint-plugin-tailwindcss/issues/241#issuecomment-1613971422) worked for me

**Comment by EliasVal** [comment]
Managed to fix the issue with [this](https://github.com/francoismassart/eslint-plugin-tailwindcss/issues/241#issuecomment-1537851714) solution!

Added this to the config:
```js
  settings: {
    tailwindcss: {
      config: path.join(__dirname, 'configs/tailwind-config/index.js'),
    }
  }
```
